src/utils/DataManipulation.js

```diff
@@ -0,0 +1,62 @@
+import { AsyncStorage } from 'react-native';
+import { filter } from 'lodash';
+import moment from 'moment';
+
+const QUALITIES = ['excellent', 'verygood', 'good', 'okay'];
+
+class DataManipulation {
+    formatData(data, qualities = QUALITIES) {
+        if (!data) return;
+
+        // removing events that are of the incorrect quality
+        for (let date of data) {
+            date.data = filter(date.data, event => qualities.includes(event.quality));
+        }
+
+        // removing days before current days
+        const now = moment();
+        let formattedData = data;
+        if (moment(data[0].datetime).isBefore(now, 'day')) {
```

Comment:
everything in this `if` can be written as `formattedData = data.filter(date => moment(date.datetime).isSameOrAfter(now, 'day'));` which imo is easier to read and understand what's happening

Reply:
Oh good point! Resolved with 82d55bb6bd8b6311bb6f2b9ac73a5b18df423827